test/unit/alphabet/cigar/cigar_op_test.cpp

```diff
@@ -0,0 +1,44 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
```

Comment:
don't you have to include gtest.h ?
please do it anyway for consistency and in case we remove the hpp files at one point

Reply:
What is the rule on including gtest.h? There are a few tests that don't include it (aa10, aa20, aa27 for example).

Reply:
Yes in general one should try to keep cpp files "independent" of recursive headers :) 
but leave the others as is. this is :nail_care: